stardis_env3.yml

```diff
@@ -0,0 +1,78 @@
+name: stardis
+
+channels:
+  - conda-forge
+
+dependencies:
+
+  # WARNING: any change to this section must be applied to the conda-forge
+  # package recipe at https://github.com/conda-forge/tardis-sn-feedstock
+
+  - python =3.8
+  - setuptools
+  - setuptools_scm <7
+  - pip
+  - numpy =1.19
+  - scipy =1.5
+  - pandas =1.0
+  - astropy =5.0
+  - numba =0.53
+  - numexpr
+  - radioactivedecay >=0.4.12
+  - pip:
+    - "--editable=git+https://github.com/tardis-sn/tardis.git@master#egg=tardis"
```

Comment:
Why editable? Also, it should be pinned to a version that we know works.

Reply:
I'm actually unable to install tardis using conda env file, that's why I have done it like this

Reply:
Updated the repo link to use https://github.com/tardis-sn/tardis/tree/release-2023.06.18 tag